pages/demos/clipboard/index.js

```diff
@@ -15,26 +15,61 @@ import { isSupported, performCopy, performPaste } from 'web-apis/clipboard';
 // demo info by id
 import { getDemoById } from 'utils/data/data-access';
 
+// Reusable component for Input Field wrapper.
+const InputFieldWrapper = ({ children }) => {
```

Comment:
How about move them in a component directory instead and import? That way future UIs can make use of it?

Here? https://github.com/atapas/webapis-playground/tree/master/components/util

Reply:
Yes, we can. 

I was thinking to move it to the `components` directory before but didn't move because the styles are tightly coupled with the UI related to this particular feature design.  The components are reusable only in the scope of this feature. That's why didn't moved. 

We can create a subfolder inside of `components/core` and name it `form`. And shift these components to this `form` directory. What do you suggest? :smile:

Reply:
Agree

Reply:
@atapas required changes are done. Please review and let me know if anything is required :+1: